Libraries/Connect/Interfaces/ConnectError.swift

```diff
@@ -73,6 +73,23 @@ public struct ConnectError: Swift.Error, Sendable {
             case payload = "value"
         }
 
+        public init(from decoder: Swift.Decoder) throws {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+
+            // Read the base64-encoded payload and then pad if needed
+            let encodedPayload = try container.decode(String.self, forKey: .payload)
```

Comment:
Should this be a `decodeIfPresent`? This is changing behavior to throw if `payload` is omitted

Reply:
Definitely should. Fixed.